openshift/che-monitoring.yaml

```diff
@@ -154,5244 +130,24 @@ objects:
-                "text": "PS Survivor Space",
-                "value": "PS Survivor Space"
-              }
-            },
-            "seriesOverrides": [],
-            "spaceLength": 10,
-            "stack": false,
-            "steppedLine": false,
-            "targets": [
-              {
-                "expr": "jvm_memory_used_bytes{application=\"$application\", instance=\"$instance\", id=\"$jvm_memory_pool_heap\"}",
-                "format": "time_series",
-                "hide": false,
-                "interval": "",
-                "intervalFactor": 2,
-                "legendFormat": "used",
-                "metric": "",
-                "refId": "A",
-                "step": 1800
-              },
-              {
-                "expr": "jvm_memory_committed_bytes{application=\"$application\", instance=\"$instance\", id=\"$jvm_memory_pool_heap\"}",
-                "format": "time_series",
-                "hide": false,
-                "interval": "",
-                "intervalFactor": 2,
-                "legendFormat": "commited",
-                "metric": "",
-                "refId": "B",
-                "step": 1800
-              },
-              {
-                "expr": "jvm_memory_max_bytes{application=\"$application\", instance=\"$instance\", id=\"$jvm_memory_pool_heap\"}",
-                "format": "time_series",
-                "hide": false,
-                "interval": "",
-                "intervalFactor": 2,
-                "legendFormat": "max",
-                "metric": "",
-                "refId": "C",
-                "step": 1800
-              }
-            ],
-            "thresholds": [],
-            "timeFrom": null,
-            "timeRegions": [],
-            "timeShift": null,
-            "title": "$jvm_memory_pool_heap",
-            "tooltip": {
-              "msResolution": false,
-              "shared": true,
-              "sort": 0,
-              "value_type": "cumulative"
-            },
-            "type": "graph",
-            "x-axis": true,
-            "xaxis": {
-              "buckets": null,
-              "mode": "time",
-              "name": null,
-              "show": true,
-              "values": []
-            },
-            "y-axis": true,
-            "y_formats": [
-              "mbytes",
-              "short"
-            ],
-            "yaxes": [
-              {
-                "format": "bytes",
-                "label": null,
-                "logBase": 1,
-                "max": null,
-                "min": 0,
-                "show": true
-              },
-              {
-                "format": "short",
-                "label": null,
-                "logBase": 1,
-                "max": null,
-                "min": null,
-                "show": true
-              }
-            ],
-            "yaxis": {
-              "align": false,
-              "alignLevel": null
-            }
-          },
-          {
-            "coll
```

Comment:
No new line in the end
Also we probably need to check with SD if we would be allowed to create the 1G PVC (I guess there should not be technical issues with this since osd-monitor is using 5GB on preview and 10GB on prod) I do not know what is the provisioning process